### DISCUSSION TITLE
onAppStart -> unable to retrieve current customer {"message": "The consumer isn't authorized to access %resources.", "parameters": {"resources": "self"}

### DISCUSSION THREAD

**Discussion by dotsinspace** [troublediehard/magento-react-native#172] [Q&A]
After refactoring code and commenting your code i start getting this error.

Could you help me how can i get it resolved.

**Comment by dimaportenko** [comment]
How can I help you?

**Reply by dotsinspace** [reply]
brother i just refatctored your code and now i am struggling with 2 errors. didnt removed anything just added comments in places. fixed typos etc.

Now error i am getting:
`
onAppStart -> unable to retrieve current customer {"message": "The consumer isn't authorized to access %resources.", "parameters": {"resources": "self"}
`
and Second error is .
`
Unrecognized operator abs
`

**Reply by dimaportenko** [reply]
Most likely you broke something while refactoring. I have no idea what you did, so not sure how can I help you.

**Reply by dimaportenko** [reply]
Try to revert your changes and see if it's working.

**Reply by dotsinspace** [reply]
ok

**Reply by sanjeevyadavIT** [reply]
hi @dotsinspace can you try this for first one https://github.com/sanjeevyadavIT/magento_react_native/issues/115#issuecomment-775198370
